CONTRIBUTING.md

```diff
@@ -81,6 +82,55 @@ $ npx prettier * --write
 
 This should be done before opening a PR to keep checked-in code consistent.
 
+### Onboarding
+
+To onboard your release tests (Dashboards plugins) onto this repo test function, you can follow the next steps.
+
+0. Create a name to identify your plugin, e.g `plugin-name`
+
+The dir name shall be descriptive to identify your plugin. You can use the same name defined in build repo https://github.com/opensearch-project/opensearch-build/tree/main/scripts/components
+
+1. Move test files under `cypress/integration/plugins/<plugin-name>`
```

Comment:
instead of `Move`, shall we call it `Place`. let's don't make assumption every plugin has release test.

Reply:
Sure. Rewording.